lib/identity_cache/cached/prefetcher.rb

```diff
@@ -0,0 +1,200 @@
+# frozen_string_literal: true
+
+module IdentityCache
+  module Cached
+    module Prefetcher
```

Comment:
Is this module public API?

Reply:
Cached is a private constant, so I believe anything under it will also be private.

Reply:
👍